pkg/controllers/obj_reconcile.go

```diff
@@ -227,6 +227,7 @@ func (o *ObjReconciler) checkForDeletedSubcluster(ctx context.Context) (ctrl.Res
 }
 
 // reconcileExtSvc verifies the external service objects exists and creates it if necessary.
+//gocyclo:ignore
```

Comment:
Not a big fan of ignoring this.  The function is getting quite big now.  Any suggestions on how we could reorganize it so we don't need this?  Maybe move the code that merges expSvc into curSvc into a separate function.

BTW, I updated this function the other day, so best to merge up before doing any big changes to it.